packages/cli/src/zosfiles/-strings-/en.ts

```diff
@@ -58,6 +58,12 @@ export default {
                     EX1: "Create an empty physical sequential data set with default parameters"
                 }
             },
+            DATA_SET_BLANK: {
```

Comment:
I believe this was used in the `create data-set-blank` command definition, but seems like it no longer exists.